renku/cli/.gitlab-ci.yml

```diff
@@ -15,14 +27,3 @@ dot:
   environment:
     name: dot/$CI_COMMIT_REF_NAME
     url: $CI_PROJECT_URL/-/jobs/artifacts/$CI_COMMIT_REF_NAME/raw/graph.dot?job=$CI_JOB_NAME
-
-review_notebook:
```

Comment:
I would leave it here so one can access the URL via GitLab.

Reply:
It's a simple change to put it back in if it's needed in the future -- right now it's not being used anywhere so it shouldn't be included.